### ISSUE TITLE
The "WCAG20_Form_HasSubmit" should exclude the hidden buttons #3769

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#903]
Hidden buttons are not visible by visual users (native hidden, display:none, or visibility:hidden) or by screen reader users (native + aria-hidden). In this case, the users will not have a submit button for the form in the current snapshot of the page.  The current rule walks DOM for submit button only.  

## This ticket is for the following (select only one): 

- [ ] `New test cases`
- [ ] `Update to a current rule`
- [ ] `Helping text for a rule`
- [ ] `other`

Original issue:
![image](https://user-images.githubusercontent.com/4451128/171905168-41e1021a-d636-4821-b2b6-5e828319b3dc.png)

**Comment by aliunwala** [comment]
From triage:
SME an Dev need to work together to determine if this is still a valid rule.
